src/main/java/com/Example/BankManagement/Entity/Account.java

```diff
@@ -1,14 +1,13 @@
-package customerRegistration.coreJava.src.main.com.tutorial.projectwithMaven;
+package com.Example.BankManagement.Entity;
 
-import javax.persistence.Table;
 
-import javax.persistence.Entity;
-import javax.persistence.Id;
+import jakarta.persistence.Entity;
```

Comment:
why use jakarta when already use javax class.

Reply:
javax.persistence-api been replaced by jakarta.persistence-api in spring data jpa starter in year 2019 due to some trademark reason